### DISCUSSION TITLE
how do i rebind the power button so suspend?

### DISCUSSION THREAD

**Discussion by KiingWatermelon** [i3/i3#4923] [Q&A] [answered]
i want to make it so when i press the power button it suspends (or sleeps)
what do i add to the config file?

**Comment by Airblader** [comment, marked as answer]
It is theoretically possible to bind the power key, but not advisable. This is handled by [systemd-logind](https://www.freedesktop.org/software/systemd/man/logind.conf.html) on modern systems, and you can configure the `HandlePowerKey` behavior to be `suspend` there.

**Reply by orestisfl** [reply]
Why would that be not advisable?

**Reply by Airblader** [reply]
Because systemd is perfectly able of handling this without rebinding an important hardware button in the user-space window manager. Let me rephrase to "I personally wouldn't advise it", but of course anyone can do what they want.

**Reply by orestisfl** [reply]
I see. I thought you meant the systemd-way was not advisable.

**Reply by KiingWatermelon** [reply]
ok that worked but is there a way to make it use i3 lock when i come back?

**Reply by Airblader** [reply]
Best would be to use `xss-lock`, see [here](https://github.com/i3/i3lock#running-i3lock).

**Reply by KiingWatermelon** [reply]
Thank you very much!
